shared/teams/routes.js

```diff
@@ -57,6 +59,16 @@ const routeTree = makeRouteDefNode({
         ...makeManageChannels,
         ...makeRolePicker,
         ...makeReallyLeaveTeam,
+        openTeamSetting: {
+          children: {},
+          component: MaybePopupHoc(ConnectedMakeOpenTeamConfirm),
```

Comment:
When I first tried to make this a popup I was surprised by how hard it was. I think this is closer to how it should work. I should be able to say make X a popup without X knowing it's a popup.